hathor/event/storage/event_storage.py

```diff
@@ -46,8 +46,13 @@ def iter_from_event(self, key: int) -> Iterator[BaseEvent]:
         raise NotImplementedError
 
     @abstractmethod
-    def clear_events(self) -> None:
-        """Clear all stored events and related metadata."""
+    def reset_events(self) -> None:
+        """Reset stored events and related metadata."""
+        raise NotImplementedError
+
+    @abstractmethod
+    def reset_all(self) -> None:
+        """Reset all data."""
```

Comment:
According to the memory storage implementation, this would also disable the event queue. Is that right? If yes, add it to the docstring.

Reply:
> this would also disable the event queue

What do you mean?

Anyway, I've improved the docstrings in  https://github.com/HathorNetwork/hathor-core/pull/608/commits/79f2773a773adcbef4f5d853344e51bb66cfb4d2

Reply:
Never mind. My comment doesn't make sense for the new implementation. Thanks!